src/coreclr/pal/inc/rt/safecrt.h

```diff
@@ -131,15 +131,7 @@ typedef int errno_t; /* standard */
 /* error codes */
 #if !defined(_SECURECRT_ERRCODE_VALUES_DEFINED)
 #define _SECURECRT_ERRCODE_VALUES_DEFINED
-#if !defined(EINVAL)
-#define EINVAL          22
-#endif
-#if !defined(ERANGE)
-#define ERANGE          34
-#endif
-#if !defined(EILSEQ)
-#define EILSEQ          42
-#endif
+#include <pal.h>
```

Comment:
This include should not be needed, the palrt.h includes pal.h and then it includes the safecrt.h, so pal.h should be already included here.

Reply:
Done.